src/commcare_cloud/fab/fabfile.py

```diff
@@ -383,33 +385,47 @@ def prepare_offline_deploy():
 
 
 @task
-def setup_release(keep_days=0):
+def setup_limited_release(keep_days=1):
+    setup_release(keep_days, full_cluster='False')
+
+
+@task
+def setup_release(keep_days=0, full_cluster='True'):
```

Comment:
Going along with the example above, I think it'd be fine to _only_ let people access the `full_cluster` functionality via the `setup_limited_release` task, rather than allowing them to either do that or pass in `:full_cluster=yes` or whatever in the command line.